website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Configure dbt Cloud CLI
+id: configure-cloud-cli
+description: "Instructions on how to configure the dbt Cloud CLI"
+sidebar_label: "Configure dbt Cloud CLI"
+pagination_next: null
+---
+
+import CloudCLIFlag from '/snippets/_cloud-cli-flag.md';
+
+<CloudCLIFlag/>
+
+
+## Prerequisites
+
+- You must set up a project in dbt Cloud.
+- You must have your [personal development credentials](/docs/dbt-cloud-environments#set-developer-credentials) set for that project. The dbt Cloud CLI will use these credentials, stored securely in dbt Cloud, to communicate with your data platform.
+- You must [enroll](/docs/dbt-versions/experimental-features) in the dbt Cloud beta features. 
+	- To enroll, navigate to your **Profile Settings** and enable the **Beta** flag under **Experimental Features**.
+
+
+## Configure the dbt Cloud CLI
+
+Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt Cloud project.
+
+1. Ensure you meet the prerequisites above.
+
+2. Download your credentials from dbt Cloud by clicking on the **Try the dbt Cloud CLI** banner on the dbt Cloud homepage.
+3. Follow the banner instructions and download the config file to `~/.dbt/dbt_cloud.yml`. The config file looks like:
```

Comment:
This is correct for Mac and Linux, but should be: C:\Users\yourusername\.dbt on windows